R/geom-text.r

```diff
@@ -10,16 +10,16 @@
 #' space they occupy on the plot is not constant in data units: when you
 #' resize a plot, labels stay the same size, but the size of the axes changes.
 #'
-#' `geom_text` and `geom_label` both add a label for each row in the
+#' `geom_text()` and `geom_label()` both add a label for each row in the
```

Comment:
Add labels for each row? (Not wed to it one way or another, they both make sense)